website/source/docs/agent/options.html.markdown

```diff
@@ -576,6 +576,24 @@ Consul will not enable TLS for the HTTP API unless the `https` port has been ass
 * <a name="node_name"></a><a href="#node_name">`node_name`</a> Equivalent to the
   [`-node` command-line flag](#_node).
 
+* <a name="performance"></a><a href="#performance">`performance`</a> Available in Consul 0.7 and
+  later, this is a nested object that allows tuning the performance of different subsystems in
+  Consul. See the [Server Performance](/docs/guides/performance.html) guide for more details. The
+  following parameters are available:
+    * <a name="raft_multiplier"></a><a href="#raft_multiplier">`raft_multiplier`</a> - An integer
+      multiplier used by Consul servers to scale key Raft timing parameters. Tuning this affects
+      the time it takes Consul to detect leader failures and to perform leader elections, at the
+      expense of requiring more network and CPU resources for better performance.<br><br>A value
+      of 0, the default, means that Consul will use a lower-performance timing that's suitable for
+      [minimal Consul servers](/docs/guides/performance.html#minumum), currently equivalent to
+      setting this to a value of 5 (this default may be changed in future versions of Consul,
+      depending if the target minimum server profile changes). Above 0, higher values imply lower
+      levels of performance. Setting this to a value of 1 will configure Raft to its
```

Comment:
"Above 0, higher values imply lower levels of performance." is confusing. Consider rephrasing to "The zero value uses the default, lower values are used to tighten timing and increase sensitivity while higher values relax timings and reduce sensitivity."